docs/contributing/debug.md

```diff
@@ -45,7 +45,23 @@ outside the developer's machine and is not recommended.
       The execution will stop and wait for the IDE or Delve CLI to attach
       to the port, specified with the `DELVE_PORT` variable.
       Once the IDE or Delve CLI is attached, the execution will continue.
-   
+
+## Running integration tests with debugger attached
+
+1. Run development container with build optimizations disabled (ie. `DOCKER_DEBUG=1`) and Delve enabled:
+   ```bash
+   $ make BIND_DIR=. DOCKER_DEBUG=1 DELVE_PORT=127.0.0.1:2345:2345 shell
+   ```
+2. Inside the development container, run the integration test you want through the `make.sh` script:
+      ```bash
```

Comment:
Is the indentation correct here? (Or vice-versa, is the indentation correct in the `1.` bullet?

If the indentation is correct, you can also add newlines around the code-blocks

Reply:
(i.e., if the indentation is correct, it won't re-start the numbering if you add newlines)